config.py

```
@@ -0,0 +1,122 @@
+        self.api_base_url = (
+            api_base_url or 
+            os.getenv('COINGECKO_API_BASE_URL', 'https://api.coingecko.com/api/v3')
+        )
+        
+        self.api_key = (
+            api_key or 
+            os.getenv('COINGECKO_API_KEY')
+        )
+        
+        self.timeout = (
+            timeout or 
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)